content/podcast/Cultural-Evolution-Implications-and-whats-next.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "Joseph Henrich and Cultural Evolution: Implications and what's next"
```

Comment:
Inconsistency in using CAPs in the titles - see the last title with all words capitalised at the beginning. Minor but I would go with Caps on all words (except and, or, etc.)